azure/README.md

```diff
@@ -4,9 +4,44 @@
 # Introduction
 
 In here you can find example using our SDK for all supported components in the Azure environment.
-
 Consult the main [README](../README.md) for details on how to set up your environment to be able to run our projects.
 
+The simplest way to use these samples is to build them from the parent project.
+If for instance you would like to build and run the `aks-full-example` you could do it by running the following two 
+commands from the current directory:
+
+```bash
+mvn clean compile assembly:single -pl shared-config,aks-full-sample
```

Comment:
I think that we don't have to specify projects, or we could list two ways, as e.g we ar ein aks-full-sample dorectory, we can use this command: `mvn clean compile assembly:single -U`

Reply:
Nope, that does not work as it needs the shared-config project to be installed.

```
[ERROR] Failed to execute goal on project azure.aks.minimum.sample: Could not resolve dependencies for project com.yanchware.fractal:azure.aks.minimum.sample:jar:2.0.0: The following artifacts could not be resolved: com.yanchware.fractal:azure.shared.config:jar:1.0.0 (absent): Could not find artifact com.yanchware.fractal:azure.shared.config:jar:1.0.0 in central (https://repo1.maven.org/maven2) -> [Help 1]
```